lib/active_merchant/billing/gateways/payeezy.rb

```diff
@@ -119,27 +127,34 @@ def add_authorization_info(params, authorization)
         params[:method] = method
       end
 
+      def add_creditcard_for_tokenization(params, payment_method, options)
+        params[:apikey] = options[:apikey]
+        params[:js_security_key] = options[:js_security_key]
+        params[:ta_token] = 'NOIW'
```

Comment:
Hello! How are you?

Thanks for helping improving this particular Gateway! I am currently working on the implementation of Payeezy and I have to use the same method as well. I just wanted to share with you that the ta_token should be configurable, since that token NOIW is only accepted in a SANDBOX environment. If we want to use this tokenization feature in a LIVE environment, the ta_token should be set according to the given by FirstData in the Terminal (take a look at the documentation for that particular field https://developer.payeezy.com/payeezy-api/apis/get/securitytokens)

Hope this helps!

Reply:
Thank you, @sebastianGitDL -- will modify now.